RealmSwift-swift3.0-dev/List.swift

```diff
@@ -31,7 +31,7 @@ public class ListBase: RLMListBase {
         return descriptionWithMaxDepth(depth: RLMDescriptionMaxDepth)
     }
 
-    @objc private func descriptionWithMaxDepth(depth: UInt) -> String {
+    @objc(descriptionWithMaxDepth:) private func descriptionWithMaxDepth(depth: UInt) -> String {
```

Comment:
Does this bridge correctly without the overridden selector if you exactly match the `description(withMaxDepth:)` that the RLMArray version appears to be brdiged to Swift as?